pytorch_lightning/accelerators/gpu_accelerator.py

```diff
@@ -0,0 +1,52 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
Are we going to add this to each file?

Reply:
yes

Reply:
in a different PR as the refactors come in